test/model.test.js

```diff
@@ -5050,10 +5050,6 @@ describe('Model', function() {
       await Model.createCollection();
       const collectionName = Model.collection.name;
 
-      // If the collection is not created, the following will throw
-      // MongoServerError: Collection [mongoose_test.User] not found.
-      await db.collection(collectionName).stats();
-
```

Comment:
what is this change? is it not necessary anymore?

Reply:
I don't think that line is necessary for the test. And `stats()` is no longer a method on collections.